app/views/hyrax/dashboard/collections/_show_actions.html.erb

```diff
@@ -9,6 +9,14 @@
                   hyrax.my_works_path(add_files_to_collection: presenter.id),
                   title: t('hyrax.collection.actions.add_works.desc'),
                   class: 'btn btn-default' %>
+      <%# TODO - The presenter's solr document does not have any collection_type information.
+          I am stuck on how to fix this. %>
+      <% if presenter.collection_type_is_nestable? %>
+        <%= link_to t('hyrax.collection.actions.nest_collections.label'),
+                    hyrax.dashboard_new_nest_collection_within_path(child_id: presenter.id),
+                    title: t('hyrax.collection.actions.nest_collections.desc'),
+                    class: 'btn btn-default' %>
+      <% end %>
```

Comment:
There were two fixes in PR #1556 that may effect this...
* collection_factory now hooks all all collections to a collection_type unless specifically using `:typeless_collection`
* collection_form added in collection_type_gid to the permitted terms list